src/styles/day.scss

```diff
@@ -0,0 +1,32 @@
+@import './variables.scss';
+
+.calendar__day {
+  width: $day-size;
```

Comment:
Ensure that `$day-size` is defined in your `variables.scss` file. Undefined variables will cause compilation errors.